src/components/TheFooter.vue

```diff
@@ -77,6 +87,11 @@ import { useStatus } from '@/stores/status'
 
 const { middlewareVersion, nodeVersion } = storeToRefs(useStatus())
 const { MIDDLEWARE_URL } = useRuntimeConfig().public
+const { APP_VERSION } = useAppConfig()
+
+const APP_CHANGELOG_URL = 'https://github.com/aeternity/aescan/blob/main/CHANGELOG.md'
+const NODE_CHANGELOG_URL = 'https://github.com/aeternity/aeternity/releases'
+const MDW_CHANGELOG_URL = 'https://github.com/aeternity/ae_mdw/blob/master/CHANGELOG.md'
```

Comment:
```suggestion
const MDW_RELEASES_URL = 'https://github.com/aeternity/ae_mdw/releases'
```

I would use the releases link here